src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/TemplatesCatalogDrawerFooter.tsx

```diff
@@ -64,17 +56,14 @@ export const TemplatesCatalogDrawerFooter: FC<TemplateCatalogDrawerFooterProps>
     }
   }, [authorizedSSHKeys, loadError, namespace, updateAuthorizedSSHKeys]);
 
-  const canQuickCreate = Boolean(processedTemplate) && isBootSourceAvailable;
-  const loaded =
-    bootSourceLoaded && processedTemplateLoaded && userSettingsLoaded && loadedRHELSubscription;
+  const canQuickCreate =
+    Boolean(isAllRequiredParametersAreFullfilled(template)) && isBootSourceAvailable;
```

Comment:
Consider changing 'isAllRequiredParametersAreFulfilled' to 'allRequiredParametersAreFulfilled' or 'areAllRequiredParametersFulfilled'.